ocfweb/api/lab.py

```diff
@@ -0,0 +1,41 @@
+from django.http import JsonResponse
+from ocflib.infra.hosts import hostname_from_domain
+from ocflib.lab.stats import get_connection
+from ocflib.lab.stats import list_desktops
+
+from ocfweb.caching import cache
+from ocfweb.caching import periodic
+
+
+@cache()
+def _enumerate_desktops():
```

Comment:
i would give this a (long) TTL, or even make it periodic, as this list is not actually static (just almost static)

Reply:
I think the list of desktops will not change unless we add/delete a desktop. The situation I can think of which we need to update is when a desktop stops working (from like hardware failures), if so, is list_desktops able to detect the change?

Reply:
Right, I agree it won't change unless we add/delete a desktop. That happens sometimes :)